doc/main/intro/limitations.rst

```diff
@@ -5,12 +5,19 @@ Known Limitations
 
 This page outlines the known limitations of oneTBB to help you better understand its capabilities. 
 
+Using Debug TBB in SYCL Program
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Limitation:** When using the Debug version of oneAPI Threading Building Blocks (oneTBB) in a SYCL program compiled using the Intel(R) oneAPI DPC++/C++ Compiler, the application may crash due to two conflicting ``tbb`` and ``tbb_debug`` libraries being loaded at the same time.
```

Comment:
```suggestion
**Limitation:** The application may crash when using the Debug version of oneTBB in a SYCL program compiled with Intel(R) oneAPI DPC++/C++ Compiler. This happens because both ``tbb`` (Release version) and ``tbb_debug`` (Debug version) libraries load simultaneously, causing conflicts.